dev/travis/js/config.php

```diff
@@ -0,0 +1,360 @@
+<?php
```

Comment:
@ishakhsuvarov Could you share part of your tests configurations related to installation? Will be much easier to copy-paste(-adjust) working solution 😉